src/main/java/duke/command/CommandNewTask.java

```diff
@@ -31,4 +38,17 @@ public void execute(TaskList taskList) {
                 + ")."
         );
     }
+
+    /**
+     * Throws an exception when there is no '/' in the user input.
+     * @param input this is the user's input
+     * @throws DukeException this shows the error message and gives the format to follow
+     */
+    public void checkS(String input) throws DukeException {
```

Comment:
Please change the method name to more wholly reflect what it does. i.e. like containsBackslash() or something.

Reply:
It would make sense if it returns a boolean as well so it could maybe be abstracted into the Parser class for other developers to use as well.